### PR TITLE
Switch to HTML table for downstream error reporting

### LINKED ISSUE #1865: Fix formatting of errors in issue repository
This is an example of a reported Bodhi update failure (from https://github.com/packit/specfile/issues/194):

| dist-git branch | error |
| --------------- | ----- |
| `epel9` | ``` There is a problem with creating a bodhi update:
{"status": "error", "errors": [{"location": "body", "name": "builds", "description": "Cannot find any tags associated with build: python-specfile-0.13.1-1.el9"}, {"location": "body", "name": "builds", "description": "Cannot find release associated with build: python-specfile-0.13.1-1.el9, tags: []"}]} ``` |

---

Tasks:

- [ ] Make sure there are no newlines that would break formatting
- [ ] Extract only error description from the exception

### PR DESCRIPTION
**Pull request by kapr200**
Fixes #1865

TODO:

- [x] Write new tests or update the old ones to cover new functionality.
- [x] Update doc-strings where appropriate.
- [x] Update or write new documentation in `packit/packit.dev`.
- [x] ‹fill in›

Fixes

Related to

Merge before/after

RELEASE NOTES BEGIN

The downstream error reporting, had been switched to html.   

RELEASE NOTES END

### CODE REVIEW

File: packit_service/worker/handlers/bodhi.py [hunk 1]
```diff
@@ -327,7 +327,13 @@ def report_in_issue_repository(self, errors: dict[str, str]) -> None:
             object="Bodhi update", dist_git_url=self.packit_api.dg.local_project.git_url
         )
         for branch, ex in errors.items():
-            body += f"| `{branch}` | ```{ex}``` |\n"
+            body += (
+                "<tr>"
+                f"<td> <code>{branch}</code></td>"
+                f"<td> <pre>{ex}</pre></td>"
```
Comment (mfocko) [inline packit_service/worker/handlers/bodhi.py:333]:
```suggestion
                f"<td><code>{branch}</code></td>"
                f"<td><pre>{ex}</pre></td>"
```

File: packit_service/worker/handlers/distgit.py [hunk 2]
```diff
@@ -486,7 +486,14 @@ def run(self) -> TaskResults:
             branch_errors = ""
             for model in sorted(models_with_errors, key=lambda model: model.branch):
                 dashboard_url = self.get_dashboard_url(model.id)
-                branch_errors += f"| `{model.branch}` | See {dashboard_url} |\n"
+                branch_errors += (
+                    "<tr>"
+                    f"<td> <code>{model.branch}</code></td>"
+                    f'<td> See <a href="{dashboard_url}">{dashboard_url}</a></td>'
```
Comment (mfocko) [inline packit_service/worker/handlers/distgit.py:492]:
```suggestion
                    f"<td><code>{model.branch}</code></td>"
                    f'<td>See <a href="{dashboard_url}">{dashboard_url}</a></td>'
```

File: packit_service/worker/handlers/distgit.py [hunk 3]
```diff
@@ -886,7 +893,13 @@ def report_in_issue_repository(self, errors: dict[str, str]) -> None:
             object="Koji build", dist_git_url=self.packit_api.dg.local_project.git_url
         )
         for branch, ex in errors.items():
-            body += f"| `{branch}` | ```{ex}``` |\n"
+            body += (
+                "<tr>"
+                f"<td> <code>{branch}</code></td>"
+                f"<td> <pre>{ex}</pre></td>"
```
Comment (mfocko) [inline packit_service/worker/handlers/distgit.py:899]:
```suggestion
                f"<td><code>{branch}</code></td>"
                f"<td><pre>{ex}</pre></td>"
```

**Review by lbarcziova — APPROVED** [review]
lgtm, can you please rebase?

**Review by mfocko — APPROVED** [review]

**Pull request merged**